File: text/octave/delete/text_delete.m
#! /usr/bin/octave -qf
#
#	text_delete.m
#
#					Nov/11/2014
#
# ------------------------------------------------------------------
path (path, "/var/www/data_base/common/octave_common");
# ------------------------------------------------------------------
printf ("*** 開始 ***\n")
arg_list = argv ();
file_txt = arg_list{1};
key_in = arg_list{2};
disp (file_txt)
disp (key_in)
#
dict_aa = struct;
dict_aa = text_read_proc (file_txt)
#
if  (isfield (dict_aa, key_in))
	dict_aa = rmfield (dict_aa, key_in)
	dict_display_proc (dict_aa)
	text_write_proc (file_txt,dict_aa)
endif
#
printf ("*** 終了 ***\n")
# ------------------------------------------------------------------

File: text/nashorn/delete/text_delete.js
#! /usr/bin/env jjs
// ------------------------------------------------------------------
//
//	text_delete.js
//
//					Jan/04/2017
//
// ------------------------------------------------------------------
load ("/var/www/data_base/common/nashorn_common/text_manipulate.js")
load ("/var/www/data_base/common/nashorn_common/file_io.js")

var file_name=arguments[0]
var key_in=arguments[1]

print	("*** 開始 ***")
print	("key_in = " + key_in)

var dict_in = text_read_proc (file_name)

delete dict_in[key_in]

text_write_proc (file_name,dict_in)

print	("*** 終了 ***")
// ------------------------------------------------------------------
